client/src/main/java/com/paypal/selion/reports/runtime/Gatherer.java

```diff
@@ -41,7 +44,10 @@ static String saveGetLocation(WebDriver driver) {
                 location = driver.getCurrentUrl();
             }
         } catch (Exception exception) {
-            logger.log(Level.WARNING, "Current location couldn't be retrieved by getCurrentUrl().", exception);
+            logger.log(
```

Comment:
I decided to change the log from including the full exception stack which can eat up 20 lines of the console on every log call to just the exception name and message.

I'm still not 100% happy with the current way this is because, when we implement page object models for mobile testing just like we have Button, Link, etc for WebTest, when a user is running a flow on appium-ios for example, every app flow action such as click will cause a SeLionReporter.log call which will print a 4 line warning.  

Below is what will be added to the console with appium-android with this warning message.  4 lines.

1 thing to note: the only place i think that actually reads this 'location' value in BaseLog is from HtmlReporter, the old report.  So might be worth just logging it to finer since this is our old report.  Or perhaps we need to add this to our new Reporter since it was in our old report...

04.17.2015 00:35:08.072,[22] com.paypal.selion WARNING Current location couldn't be retrieved by getCurrentUrl(). This can be SAFELY IGNORED if testing a non-web mobile application. Reason: org.openqa.selenium.WebDriverException: Not yet implemented. Please help us: http://appium.io/get-involved.html (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 12 milliseconds
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'LM-AUN-00874810', ip: '127.0.0.1', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.9.5', java.version: '1.7.0_45'
Session ID: 8aed842a-6818-4f42-b915-bd56a2fa7c09
Driver info: com.paypal.selion.platform.grid.SeLionAppiumAndroidDriver
Capabilities [{platform=LINUX, app=/Users/antnguyen/dev/repo/SeLion/client/src/test/resources/apps/selendroid-test-app-0.15.0.apk, acceptSslCerts=true, javascriptEnabled=true, browserName=Android, mobileNodeType=appium, networkConnectionEnabled=true, unexpectedAlertBehaviour=ignore, desired={platformVersion=5.0.1, app=/Users/antnguyen/dev/repo/SeLion/client/src/test/resources/apps/selendroid-test-app-0.15.0.apk, javascriptEnabled=true, acceptSslCerts=true, platformName=Android, deviceName=Android Emulator, name=com.paypal.selion.android.sample.AppiumAndroidNativeAppTest:testWithNativeApp(), mobileNodeType=appium, unexpectedAlertBehaviour=ignore, takesScreenshot=true, version=}, locationContextEnabled=false, version=, platformVersion=5.0.1, databaseEnabled=false, platformName=Android, deviceName=Android Emulator, name=com.paypal.selion.android.sample.AppiumAndroidNativeAppTest:testWithNativeApp(), webStorageEnabled=false, warnings={}, takesScreenshot=true}]